dune-project

```diff
@@ -266,37 +288,56 @@ conduit for SSL.")
 (package
  (name cohttp-curl-async)
  (synopsis "Cohttp client using Curl & Async as the backend")
- (description "\
-An HTTP client that relies on Curl + Async for the backend. Does not require
-conduit for SSL.")
+ (description
+  "An HTTP client that relies on Curl + Async for the backend. Does not require\nconduit for SSL.")
  (depends
   ocurl
-  (http (= :version))
+  (http
+   (= :version))
   stringext
   core
   (core_unix (>= v0.14.0))
   async_kernel
   async_unix
   (cohttp-async :with-test)
-  (uri (and :with-test (>= 4.2.0)))
+  (uri
+   (and
+    :with-test
+    (>= 4.2.0)))
   (fmt :with-test)
   (ounit :with-test)
   (alcotest :with-test)))
 
 (package
  (name cohttp-bench)
  (synopsis "Benchmarks binaries for Cohttp")
- (description "\
-This package contains some benchmarks for http and cohttp.
-The benchmarks for the server latency will require wrk2
-(https://github.com/giltene/wrk2) to run. The latency graphs
-can then be generated with HdrHistogram plotter, also available
-online at https://hdrhistogram.github.io/HdrHistogram/plotFiles.html.")
+ (description
+  "This package contains some benchmarks for http and cohttp.\nThe benchmarks for the server latency will require wrk2\n(https://github.com/giltene/wrk2) to run. The latency graphs\ncan then be generated with HdrHistogram plotter, also available\nonline at https://hdrhistogram.github.io/HdrHistogram/plotFiles.html.")
  (depends
-  (core (>= v0.13.0))
+  (core
+   (>= v0.13.0))
   core_bench
-  (http (= :version))
-  (cohttp (= :version))
-  (cohttp-lwt-unix (= :version))
-  (cohttp-server-lwt-unix (= :version))
-  (cohttp-async (= :version))))
+  (http
+   (= :version))
+  (cohttp
+   (= :version))
+  (cohttp-lwt-unix
+   (= :version))
+  (cohttp-server-lwt-unix
+   (= :version))
+  (cohttp-async
+   (= :version))))
+
+(package
+ (name cohttp-eio)
+ (synopsis "CoHTTP implementation with eio backend")
+ (description
+  "A CoHTTP server and client implementation based on `eio` library. `cohttp-eio`features a multicore capable HTTP 1.1 server. The library promotes and is built with direct style of coding as opposed to a monadic.")
+ (depends
+  base-domains
+  eio
+  cstruct
+  bigstringaf
+  fmt
+  (http
```

Comment:
```suggestion
  (eio_main :with-test)
  (http
```

Reply:
Is there a way to run the tests without `ncat`? Otherwise we need to add `conf-ncat` to `opam-repository` and add it among the test dependencies

Reply:
Done.

Reply:
Not at the moment, `ncat` in the nmap is the only cross platform `ncat` that works consistently across macos and linux. I will send a pr to create `conf-nmap` in the opam-repository shortly.

Reply:
Ok, perfect, thanks!